frontend/packages/operator-lifecycle-manager/src/components/ClusterServiceVersionLogo.tsx

```diff
@@ -0,0 +1,55 @@
+import * as React from 'react';
```

Comment:
Moved this component to a separate file due to circular dependency error: `Cannot access 'HorizontalNavFacade' before initialization `

Reply:
Nit: the existing package uses lowercase letters and hypens for filenames.  Would be good to be consistent.